askbot/templates/user_profile/user_network.html

```diff
@@ -7,7 +7,7 @@
 {% block usercontent %}
```

Comment:
Should this section get the .user-profile-page body tag?  It seems like this is the only profile page tab that has the tab bar below the users name